copulas/multivariate/vine.py

```diff
@@ -207,7 +207,10 @@ def train_vine(self, tree_type):
             LOGGER.debug('finish building tree: {0}'.format(k))
 
     def get_likelihood(self, uni_matrix):
-        """Compute likelihood of the vine."""
+        """Compute likelihood of the vine.
+
+        TODO: explain what this is supposed to do and make it work
```

Comment:
What is this function doing? There's a test case for it but I have no idea what it does and why it does it. Apparently uni_matrix is supposed to be a 1d array?